### PR TITLE
use solidity naming convention

### PR DESCRIPTION
**Pull request by joshualyguessennd**

### CODE REVIEW

**Comment by jaketimothy** [comment]
Task linked: [CU-8686h7h1n Update Contract Naming Conventions](https://app.clickup.com/t/8686h7h1n)

File: src/dividend/XdShare.sol [hunk 1]
```diff
@@ -11,43 +11,43 @@ import {ReentrancyGuardUpgradeable} from
...
+ *      It serves as a reinvestment token that uses DShare as the underlying token.
  *      Additionally, it employs the ERC4626 standard for its operations.
- *      If TokenManager is not used, make sure that dShare will never split.
- * @author Dinari (https://github.com/dinaricrypto/sbt-contracts/blob/main/src/xdShare.sol)
+ *      If TokenManager is not used, make sure that DShare will never split.
+ * @author Dinari (https://github.com/dinaricrypto/sbt-contracts/blob/main/src/XdShare.sol)
  */
 // slither-disable-next-line missing-inheritance
-contract xdShare is Initializable, ERC4626, OwnableUpgradeable, ReentrancyGuardUpgradeable {
+contract XdShare is Initializable, ERC4626, OwnableUpgradeable, ReentrancyGuardUpgradeable {
```
Comment (jaketimothy) [inline src/dividend/XdShare.sol:22]: `XDShare`

**Review by jaketimothy — CHANGES_REQUESTED** [review]
Add slither detector for `naming-convention`

File: src/dividend/XdShare.sol (hunk 1 above)
Comment (joshualyguessennd) [reply src/dividend/XdShare.sol]: detect `naming-convention` it will flag for variable like _perOrderFee, starting with _

**Review by jaketimothy — APPROVED** [review]

**Pull request merged**